pkg/security/probe/probe_bpf.go

```diff
@@ -258,11 +267,7 @@ func (p *Probe) handleEvent(CPU int, data []byte, perfMap *manager.PerfMap, mana
 	offset += read
 
 	eventType := EventType(event.Type)
-
-	// zero the event first
-	event.Zero(eventType.String())
-
-	log.Tracef("Decoding event %s", eventType)
+	log.Tracef("Decoding event %s", eventType.String())
```

Comment:
nit: isn't the `%s` format already making sure that the `String` function is called ?

Reply:
it is re-added by the revert :) anyway I fixed it in the following commit